CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ endif()
 option(USE_CLANG_TIDY "use clang-tidy" OFF)
 option(LINT_AS_ERRORS "clang-tidy warnings as errors" OFF)
 
+option(USE_SANITIZER "use sanitizer" OFF)
```

Comment:
Could you add more information in the message about what sanitizer it is?  And in the code comments, write down the reference and/or URL for the sanitizer.